modules/common/locals.tf

```diff
@@ -5,4 +5,5 @@ locals {
   ingress_domain = var.domain != null ? "${var.environment_name}.${var.domain}" : null
 
   storage_class_name = "efs-cs"
+  create_shared_home = var.shared_home_size != null
```

Comment:
It looks more straightforward if we create a new `create_shared_home` flag and pass from root level rather than checking shared_home_size.